src/averageAges.js

```diff
@@ -37,7 +45,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const milfs = people.filter(milf => {
```

Comment:
```suggestion
  const milfs = people.filter(milf => {
```
use meaningful names for variables, I believe if you will apply for a job in this special industry, you will have the opportunity to use such names, but now, better replace it with another